fix:Troubleshooting - Deployment Errors - Email (#682)
appsmithorg/appsmith-docs: website/docs/help-and-support/troubleshooting-guide/deployment-errors.md

2. Run `docker-compose up -d`

:::tip

> To Stop a previous version of appsmith running on these ports, run the following:

Reviewer: ```suggestion
To stop a previous version of appsmith running on these ports, run the following:
```

* ```sudo su```
* ```docker container kill $(docker ps -q)```
:::

## Containers Failed to Start

If you chose to initialize a new database and are seeing this error, it could be due to an error while fetching dependencies during installation. Deleting the current installation direction, killing the docker containers, and restarting the installation should work. If it doesn't, please reach out to us on [discord](https://discord.com/invite/rBTTVJp)

If you are trying to connect to an existing [MongoDB](../../reference/datasources/querying-mongodb/) and the containers failed to start it may be due to one of the following reasons:

1. Incorrect [MongoDB](../../reference/datasources/querying-mongodb/) credentials
2. Empty Salt / Password for encryption

Restart the installation process with valid values.

## Unable to access Appsmith

* Ensure your security groups are configured to allow traffic to ports 80 & 443 on your installation instance.
* You can access the running application on[ localhost](http://localhost) in any browser or the `public IP` of your machine.

> * You may need to wait for few minutes before accessing the application to allow [Nginx](https://www.nginx.com) to start.

Reviewer: ```suggestion
* You may need to wait for a few minutes before accessing the application to allow [Nginx](https://www.nginx.com) to start.
```

APPSMITH_MONGODB_URI=mongodb://<your_username>:<your_password>@mongo/appsmith?retryWrites=true
```

> Change it to the following (notice the only change is `&authSource=admin`. Don't copy-paste this whole line. Only add the `&authSource=admin` part to your existing value.

Reviewer: ```suggestion
Change it to the following (notice the only change is `&authSource=admin`. Please don't paste this whole line. Only add the `&authSource=admin` part to your current value.
```

```

In a minute or two, the server should now come up and be ready.

> ## Email errors

Author: done

> If you are using Docker and are having trouble with invitation emails not being sent, even though the Admin email setup is able to send test emails, it may be due to an issue with the configuration of the Docker container.

Reviewer 2: I don't think this is a Docker-specific error. It was because the APPSMITH_REPLY_TO and APPSMITH_MAIL_FROM had to have the same email.  This could be same for Kubernetes too.  So I think we should start with 

"If you are having trouble with invitation emails not being sent...... with the configuration of the email parameters"

Reviewer 2: @ankitakinger can you please ocnfirm the above? Thanks :)

Reviewer 3: @danciaclara You're right. This is not docker related error. Sharing the link which explains what was the issue that was causing this:
https://theappsmith.slack.com/archives/C02MUD8DNUR/p1652682757105139?thread_ts=1652681174.350099&cid=C02MUD8DNUR

> If you aren't receiving the invitation email, please check the value for ```APPSMITH_REPLY_TO``` in your **docker.env** file. If this value is empty, please set it to the same email address you are using for ```APPSMITH_MAIL_FROM``` and **restart the application**.

Reviewer 2: If the above comment is true then instead of docker.env let's say "in the configuration file"

> If you aren't receiving the invitation email, please check the value for ```APPSMITH_REPLY_TO``` in your **docker.env file**. If this value is empty, please set it to the same email address that you are using for ```APPSMITH_MAIL_FROM``` and **restart the application**.

Reviewer: ```suggestion
If you aren't receiving the invitation email, please check the value for ```APPSMITH_REPLY_TO``` in your **docker.env** file. If this value is empty, please set it to the same email address you are using for ```APPSMITH_MAIL_FROM``` and **restart the application**.
```

> This should resolve the issue with not receiving the invitation email. Additionally, it may be helpful to verify that the email server being used is working correctly and that there are no issues with the network or other components that could be preventing the emails from being sent.

Reviewer: ```suggestion
This should resolve the issue of not receiving the invitation email. Additionally, it may be helpful to verify that the email server being used is working correctly and there are no issues with the network or other components preventing the emails from being sent.
```
